### DISCUSSION TITLE
tem showcase da delegua?

### DISCUSSION THREAD

**Discussion by ghost** [DesignLiquido/delegua#683] [P&R] [answered]

**Comment by ItaloCobains** [comment, marked as answer]
Se não me engano não tem. 

O que vocês acham? @leonelsanchesdasilva @samuelrvg

**Reply by leonelsanchesdasilva** [reply]
@stokebreakup @ItaloCobains O mais próximo disso é o [delegua-web](https://designliquido.github.io/delegua-web), mas acho a ideia maneira. Podemos discutir aqui o projeto e começar alguma coisa.

**Comment by ItaloCobains** [comment]
@stokebreakup poderia dar alguns exemplos de websites desse modelo?

**Reply by ghost** [reply]
O @ItaloCobains. tenho vários links oficiais de várias linguagens de programação/frameworks que tem isso, mas vou citar apenas um para não parecer spam: `https://www.builtwithelm.co/`

**Reply by ItaloCobains** [reply]
@leonelsanchesdasilva acho que isso da para ser implementado em [delegua-web](https://designliquido.github.io/delegua-web)

**Reply by leonelsanchesdasilva** [reply]
@ItaloCobains Mais ou menos. Delégua não tem exatamente uma interface gráfica. O que penso é algo misturado com delegua-web e o GitHub Gist: uma espécie de centralizador de receitas de código. 

Em teoria não é difícil fazer. Acho que poderíamos definir um escopo inicial e ir trabalhando numa primeira implementação. Dá pra usar a ideia de delegua-web, em que usamos o Mônaco Editor + uma interface de entrada e saída como a da extensão, em que simulamos um terminal na página.

**Reply by ItaloCobains** [reply]
@leonelsanchesdasilva entendi seu ponto, porem tenho algumas dúvidas, qualquer pessoa pode publicar seu trecho de código? Ou terá algum sistema de login para guardar informações daquela pessoa que ta colocando seu gist?

**Reply by leonelsanchesdasilva** [reply]
@ItaloCobains Podemos manter uma estrutura super simples apenas com as referências registradas, mas não o conteúdo em si.
